SampleCode/SwiftSampleCode/DronecodeSDKSwiftDemo/Manager/CoreManager.swift

```diff
@@ -49,7 +50,7 @@ class CoreManager {
     }
     
     public func start() -> Void{
-        core.connect()
+       _ = core.connect().toBlocking().materialize()
```

Comment:
There's certainly a nicer way without blocking, but that will do it for now, as it fixes the build :).